frontend/document/document.lua

```diff
@@ -209,6 +210,7 @@ function Document:getProps(cached_doc_metadata)
         language     = language,
         keywords     = keywords,
         description  = description,
+        identifiers  = identifiers
```

Comment:
Sidenote, but we prefer trailing commas (both for less effort when adding something and to at least sometimes have a single-line diff).